src/mixins/dom.js

```diff
@@ -0,0 +1,34 @@
+// DomMixin
+//  ---------
+
+import Backbone from 'backbone';
+
+export default {
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  appendChildren(el, children) {
+    Backbone.$(el).append(children);
+  },
+
+  beforeEl(el, sibling) {
+    Backbone.$(el).before(sibling);
+  },
+
+  replaceEl(newEl, oldEl) {
+    Backbone.$(oldEl).replaceWith(newEl);
```

Comment:
![NOOOOOOooooOOOOOOooOOOO!!!!!](http://i0.kym-cdn.com/photos/images/original/000/016/602/TLI3YSU5TZI4GJDDH3OU2TNC4FXYME6I.gif)

ah right. will update.
Gonna have to come up for a test for that some day.